src/view.rs

```diff
@@ -45,8 +45,12 @@ pub fn handle_event<'a>(event: &Event, state: &'a State) -> Option<SpacerunEvent
                         if let Command::Node(command_node) = &state.selected_command {
                             let found_child =
                                 command_node.children.iter().find(|&child| match child {
-                                    Command::Node(child_node) => child_node.key == virtual_keycode,
-                                    Command::Leaf(child_leaf) => child_leaf.key == virtual_keycode,
+                                    Command::Node(child_node) => {
+                                        child_node.shortcut == (&input.modifiers, &virtual_keycode)
```

Comment:
Yeah, you could write `let shortcut = Shortcut { key_code: virtual_keycode, modifiers: input.modifiers }` on top here instead?